tests/test_init.py

```
@@ -2051,6 +2051,56 @@ async def test_migrate_4_to_5_defers_when_firewall_rules_unavailable(
+    entry_prefix = slugify("device-id")
+    legacy_filter = _RegistryEntity("switch.filter", f"{entry_prefix}_filter_123")
+    native_firewall = _RegistryEntity(
+        "switch.firewall_rule", f"{entry_prefix}_firewall_rule_current"
+    )
+    native_nat = _RegistryEntity(
+        "switch.firewall_nat", f"{entry_prefix}_firewall_nat_source_nat_current"
+    )
+
+    entity_registry = MagicMock()
+    entity_registry.async_remove = MagicMock()
+    monkeypatch.setattr(er, "async_get", lambda hass: entity_registry)
```

Ruff (0.15.21)

[warning] Unused lambda argument: `hass`

(ARG005)